spring/pom.xml

```diff
@@ -22,33 +22,45 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <name>Transformation :: API :: Java</name>
+    <name>Transformation Rest Spring Stubs</name>
```

Comment:
```suggestion
    <name>Transformation :: API :: Rest Spring Stubs</name>
```

Reply:
My thought is that, if the artifact name will have "api" in it then the name of the module should too.